### PR TITLE
Add event backend

### PR DESCRIPTION
**Pull request by kemalfidan**
This one adds to the KKB-33 get events PR I made earlier too. I created the api and server functions for add event. The structure is very basic for now, so it doesn't upload images or have auth yet. Here's what the POST request with the form data looks like in case you wanted to look at it:
Screen Shot 2021-02-14 at 8 02 27 PM

### CODE REVIEW

**Review by rossketron — APPROVED** [review]
Looks good to me!

File: server/models/Event.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    },
+    hours: {
+        type: String,
+        required: true,
+    },
+    image: {
+        type: ContentfulImageSchema,
+        required: false,
+    },
+});
```
Comment (andlrutt) [inline server/models/Event.ts:42]: Do we need to add the signed up attendees, present attendees, and absent attendees properties?

**Review by andlrutt — APPROVED** [review]
Looks good!

File: server/models/Event.ts (hunk 1 above)
Comment (kemalfidan) [reply server/models/Event.ts:42]: We do. I haven't finalized how we should store them yet (maybe _ids of attendees, maybe denormalized to only hold the info we need from the volunteers type). Definitely will need to get updated when we figure it out though,

**Comment by rluberto** [comment]
LGTM

**Pull request merged**